lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -38,7 +38,7 @@ const PolygonParameterEditor = React.createClass({
     },
 
     getValue() {
-        const pointsLongLats = this.props.parameterValues[this.props.parameter.id];
+        const pointsLongLats = this.props.parameterValues[this.props.parameter.id][0];
```

Comment:
Ah! That explains it. However I don't think any of the `jsx` files are setting the defaultValue for any parameters - the only place (apart from your change in #1910) is in the `Models/*Parameter.js` files. And there, the jsdocs say it is a Boolean.  I would suggest changing the jsdoc info. :-)